### PR TITLE
Citations import fix NULL authors

### PR DESCRIPTION
**Pull request by dglemos**

### CODE REVIEW

File: modules/t/publicationAdaptor.t [hunk 1]
```diff
@@ -77,6 +77,21 @@ $va->db->include_failed_variations(1);
 my $varfs = $va->fetch_all_by_publication($pubs->[0]);
 ok(scalar(@{$varfs}) ==2,   "variation count by publication - inc fails");
 
+## store NULL authors
```
Comment (helensch) [inline modules/t/publicationAdaptor.t]: Suggest comment modified to say what testing e.g. authors stored as NULL when not  provided. 
Includes "" and 0.

File: modules/t/publicationAdaptor.t [hunk 2]
```diff
@@ -77,6 +77,21 @@ $va->db->include_failed_variations(1);
...
+my $pub_store_null = Bio::EnsEMBL::Variation::Publication->new(
```
Comment (helensch) [inline modules/t/publicationAdaptor.t]: The test is similar to the other store test. If the database is left intact between test runs, the records are kept.
For future suggest considering using hide and restore e.g. in motifFeatureVariationAdaptor.t

Comment (dglemos) [reply modules/t/publicationAdaptor.t]: You mean using hide and restored for just this new test?

Comment (helensch) [reply modules/t/publicationAdaptor.t]: Yes - it was something to consider for future. - Do we need to do this generally for tests and are DBAdaptors different. I'll add a ticket for.

**Comment by helensch** [comment]
merged into master and release/101

**Pull request merged**